src/main/java/ch/wisv/areafiftylan/service/CurrentUserServiceImpl.java

```diff
@@ -63,4 +88,19 @@ public boolean canAccessOrder(Object principal, Long orderId) {
             return false;
         }
     }
+
+    @Override
+    public boolean hasTicket(Object principal) {
```

Comment:
I'd rename this to hasAnyTicket. hasTicket seems as if you'd have a specific ticket in mind. At least that was my first understanding when reading the name in the TeamRestController.